src/DemoUI/src/Controllers/ResolverController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Packlink\DemoUI\Controllers;
+
+use Packlink\BusinessLogic\DTO\Exceptions\FrontDtoValidationException;
+use Packlink\DemoUI\Controllers\Models\Request;
+
+/**
+ * Class ResolverController
+ *
+ * @package Packlink\DemoUI\Controllers
+ */
+class ResolverController
+{
+    /**
+     * Main entry point. In this method the request will be resolved.
+     */
+    public function handleAction()
```

Comment:
Codacy found an issue: [handleAction accesses the super-global variable $_GET.](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)